versioned_docs/version-1.3.0-beta.1/how_to_guides/multiarch.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Multi-Architecture Image/Artifact Support
+sidebar_position: 110
+---
+
+# Multi-Architecture Image/Artifact Support
+
+ORAS can help you create and manage multi-architecture artifacts such as configurations, executables, and container images.
+This example shows how you can use ORAS to manage executables for different operating systems and architectures.
+
+## Pushing the artifacts
+The first step is to push the artifact with platform information to your container registry.
+
+First we push a Linux ARM64 executable:
+```console
+$ oras push --plain-http localhost:15000/oras:linux_arm64 --artifact-platform linux/arm64 --artifact-type 'application/vnd.oci.image.config.v1+json'  bin/linux/arm64/oras:application/octet-stream
```

Comment:
Do we want to put specify a more meaningful artifact type? The artifact to push is not an image config. 

BTW, Is the `--plain-http` flag necessary here? I feel that it makes the command too long.

Reply:
Anyone have thoughts on what would be a better artifact-type here? I thought it was an image config containing `application/octet-stream`.

Reply:
Should the artifact type refer to the type of `bin/linux/arm64/oras`?

Reply:
Got rid of `--plain-http` I don't think artificate type should refer to `application/octet-stream` that is the type of the artifact itself

Reply:
How about something like `"application/vnd.example` 😂